aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/befta/DefinitionStoreTestAutomationAdapter.java

```diff
@@ -7,7 +7,7 @@
 
 public class DefinitionStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
 
-    public static final String DEFAULT_DEFINITIONS_PATH = "uk/gov/hmcts/befta/dse/ccd/definitions/";
+    public static final String DEFAULT_DEFINITIONS_PATH = "uk/gov/hmcts/befta/dse/ccd/definitions/valid";
```

Comment:
all latest files are moved to definitions/valid folder in this one, for separating the brand-new invalid files that are added to the fw, but this is not the point in this change to fix the build.